phpunit-multisite.xml.dist

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" ?>
+<phpunit
+  bootstrap="tests/phpunit/bootstrap.php"
+  backupGlobals="false"
+  colors="true"
+  convertErrorsToExceptions="true"
+  convertNoticesToExceptions="true"
+  convertWarningsToExceptions="true"
+>
+	<php>
+		<ini name="error_reporting" value="32767" />
+		<ini name="display_errors" value="1" />
+		<ini name="display_startup_errors" value="1" />
+		<const name="WP_TESTS_MULTISITE" value="1" />
+		<const name="GOOGLESITEKIT_PLUGIN_MAIN_FILE" value="1" />
```

Comment:
Also needs to be done in `phpunit.xml.dist`

```suggestion
```

Reply:
This was a copy and paste from phpunit.xml.dist. Should it be remove there well.

Reply:
Yes, as I said in my comment, this also needs to be done in `phpunit.xml.dist` 🙂

Reply:
Resolved in https://github.com/google/web-stories-wp/pull/2534/commits/e1f627bcf00082964f663cb2a695fc185babe38c